java/src/apps/gui/GuiLafPreferencesManager.java

```diff
@@ -293,7 +293,7 @@ public void applyLookAndFeel() {
      * for each.
      */
     private void applyFontSize() {
-        logAllFonts();
+        if (log.isDebugEnabled()) logAllFonts();
```

Comment:
This needs to be isTraceEnabled or you are going to be reading the code to understand why it's not logging at the debug level.